### PR TITLE
Remove brigmedic

### PR DESCRIPTION
**Pull request by EmoGarbage404**
## About the PR

as decided in the extensive thread in discord, brigmedic doesn't really need to exist. In the interest of not making emisse kill me, i'm just using migrations instead of deleting all the files.

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- remove: Removed brigmedic.

### CODE REVIEW

File: Resources/migration.yml [hunk 1]
```diff
@@ -9,3 +9,25 @@
 
 chem_dispenser: ChemDispenser
 
+# brigmedic stuff. Remove this sometime soon.
+SpawnPointBrigmedic: null
+ClothingHeadsetBrigmedic: null
```
Comment (metalgearsloth) [inline Resources/migration.yml:14]: I'm adding drozd atm and I think it would be beneficial to date these, I just used # 2023-05-04.

**Review by metalgearsloth — APPROVED** [review]

File: Resources/migration.yml (hunk 1 above)
Comment (0x6273) [reply Resources/migration.yml]: Or even better, split it into multiple files prefixed with their date. Then there won't be merge conflicts.

**Comment by PuroSlavKing** [comment]
> > if brigmedic will had nerf it will be boring role, without any specific gameplay function
> 
> I disagree, but most of my comments on it are on the discord. I don't really want to put additional input here other than I disagree with 
> > if brigmedic will had nerf it will be boring role, without any specific gameplay function
> 
> I disagree, but most of my comments on it are on the discord. I don't really want to put additional input here other than I disagree with it being removed, and still find it to be a useful role.

 The proposal for the sake of raising an informational occasion, there are no specific arguments for removing the whole role, you can just as well remove a zoo technician, a boxer and others. He has nothing to do, and he decided to remove the role, it would be better to do useful things. 
 The only useful thing that can be taken out of the dispute in discord is to remove the brigmedic's hardsuit to the armory and remove the stunner because of the LRP of the guys.

**Comment by Yayts0** [comment]
I'll just leave what I said in the thread on the discord.
Brigmed is necessary because:
- You got a person in your brig area that you can 100% trust isn't a syndie
- You put less on sec's shoulders by dividing the responsibility of treating the wounded
- Sec has a second person that can watch the department, making it so playing FNAF as warden can't end up so badly
- You don't have to hope and pray for a paramed to come by your department when calling one on comms (I've had many experiences where paramed is called and legit none come)
- This goes back to a previous point but this time for med, med won't have to worry that much for security since they have their brigmedic to cover em. so as medical you got less to worry about.
To put simply, I don't agree with its removal.

**Comment by luckyshotpictures** [comment]
They have to be prepared for any situation that may come up in the brig. Spacings, Zombies, Tarantulas, Slimes. Maybe even a damn Revenant. I don't see why the Brigmedic should be restricted from using hardsuits. Keep in the armory if this is too much of a problem IMO, but don't remove the hardsuit. Plus, it looks really nice. I disagree with those who argue that the Brigmedic hardsuit is too strong. While it does provide added protection, a skilled opponent can easily take down a Brigmedic if they're not careful.

I find it incredibly helpful to have access to medical supplies as a Brigmedic from roundstart, and that I don't have to go on a wild easter egg hunt for supplies. It allows me to treat any injuries that may occur in the brig early without having to rely on other medical staff, which saves time and ensures that prisoners and officers receive timely treatment. And let's face it - injuries are common in the brig, so having a medical doc on site is essential.

In my experience, the Brigmedic role has been incredibly beneficial for both security officers and prisoners. It saves time and stress for officers by allowing them to quickly treat minor injuries, and it provides a sense of security for prisoners who may be injured or ill. Removing this role would put additional weight on the Warden and HoS, which could make intense shifts even more challenging. The Brigmedic turns to a combat medic when Nukies show up, or Zombies try to overrun Security.  It's not something that your regular Joe Schmo Paramedic can handle.

If a Brigmedic keeps hanging out in medical, it's simply an in-character issue that should be dealt with accordingly, not the issue of the role itself.

Overall, I think that the Brigmedic is a valuable addition to the game, and it enhances the gameplay experience for both security and medical staff. It takes away from the "Oh shit, this doc could betray me, I have to be careful." and gives a little more relaxed atmosphere. Which, is very necessary on LRP, where things can get intense so fucking fast.

**Comment by SEMoslaw** [comment]
Shame to see brigmed go I really enjoyed the role and I think if the problem were people powergaming you should just remove a bit of access, disabler and move his hardsuit to armory.

**Comment by Skyedra** [comment]
As someone who plays a lot of sec, I'd really like to see this reverted and brigmedic return.  It's very helpful to have a healer on team who is dedicated to sec, given how much damage sec takes.  Kinda like how nukies get their own healer, sec also needs a focused one.

**Comment by deltanedas** [comment]
you could remove the hardsuit or move it into the armoury and remove 90% of the stuff in brigmeds locker and itd still be a good role

**Comment by Skyedra** [comment]
One more point in advocating revert -- the removal seems unwelcome by more players than not:

image

image

**Comment by Nutjob** [comment]
I'm one of those on Discord that advocated against the removal, and I'm also asking to reconsider because it was removed REALLY fast, with several people against it.
I'll paste here what I said there:

- Brigmed makes being in Security REALLY much less stressful, not only for minor injuries on sec officers, but for arrested individuals and prisoners in general.
- Having to drag heavily injured prisoners to medbay was a giant pain, because they almost always were in critical conditions from trying to break out (and harming officers).
- Using Paramedics as substitutes is not that efficient because of the lack of permissions, let alone if they're not already busy trying to save people in the station or in space (Normal medics are already hands-full in medbay).
- The Brigmed players are usually already knowledgeable on medical procedures, so you almost always have a guarantee that the situation will be handled in a decent way.
- Removing the Brigmed role will put lots of additional weight on the Warden and HoS, which on intense shifts can be a literal hell if one of the 2 is busy/missing/ssd.

In short, please keep the job; if you're afraid of powergaming you can:
- Nerf his equipment (like removing the disabler).
- Less permissions (like having only sec and sec medroom, not medbay).
- Role equipment (like the hardsuit) in Armory, given only on discretion of Warden and HoS. 
- Or, make it like a loadout job that can be granted by HoP on HoS/Captain orders.

Again, please reconsider this decision.
